policies/attribute_name_collisions.rego

```diff
@@ -41,6 +41,9 @@ deny contains attr_registry_collision(description, name) if {
     name := attribute_names[i].name
     prefix := attribute_names[i].namespace_prefix
 
+    # ignore otel.scope attribute
+    name != "otel.scope"
```

Comment:
> I believe we can/should fix collision policy for template attributes 

@lmolkova, @joaopgrassi, do you have any other preference how to make changes in the collision policy?

Reply:
Maybe we just exclude template attributes from the namespace check? https://github.com/open-telemetry/semantic-conventions/blob/8638e7b4b9ff38da58559e04ef63172cca04ed0a/policies/attribute_name_collisions.rego#L39

But we will probably need to add the `type` property to https://github.com/open-telemetry/semantic-conventions/blob/8638e7b4b9ff38da58559e04ef63172cca04ed0a/policies/attribute_name_collisions.rego#L6

Reply:
Yes, it has to be either done as part of this PR or before, otherwise we can never merge this as the check will fail. I'd prefer to not go with the current state and implement the proper exclusion.

Reply:
@joaopgrassi, are you able to help and create a PR for the proper exclusion?

Reply:
Let's see what @lmolkova thinks, I have to confess I'm not so up to speed with all the rego rules, haven't written one myself yet :D. But yeah if that's the direction we agree I can do that sure :)